common/mod/vecevent.mod

```diff
@@ -0,0 +1,163 @@
+:  Vector stream of events, copied from nrn/share/examples/nrniv/netcon/vecevent.mod
+
+COMMENT
```

Comment:
I guess this is just a copy?

Reply:
Yes, exactly copy of the `nrn/share/examples/nrniv/netcon/vecevent.mod`